src/app/components/quick-connect-dialog/quick-connect-dialog.component.ts

```diff
@@ -46,6 +53,7 @@ export class QuickConnectDialogComponent implements OnInit, OnDestroy {
     suggestions: SuggestedConnection[] = [];
     loading = false;
     pfsError = false;
+    noPfs = false;
```

Comment:
Negated variables can cause awkward comparisons. `if (PfsConfigured)` is usually easier to read than `if (!noPfs).`